src/Uno.UI.Tests/Windows_UI_Xaml_Data/BindingTests/Given_Binding.cs

```diff
@@ -139,7 +141,29 @@ public void When_Xaml_Object_With_Common_Properties()
 
 			SUT.ForceLoaded();
 
-			Assert.AreEqual("42", SUT.topLevel.Text);
+			SUT.topLevel.Should().NotBeNull();
+			SUT.topLevel.Text.Should().Be("42");
+
+			var bindingExpression = SUT.topLevel.GetBindingExpression(TextBlock.TextProperty);
+			bindingExpression.Should().NotBeNull();
+
+			var binding = bindingExpression.ParentBinding;
+			binding.Should().NotBeNull();
```

Comment:
You can use `AssertionScope` to get all errors at once.

```suggestion
            using var _ = new AssertionScope()

			binding.Should().NotBeNull();
```

Reply:
Thank you, I didn't know about that. 
I need to test the behavior in case there is an NRE here `binding.Path.Should().NotBeNull();` if binding is null

Reply:
There seems to be a special handling for NRE that will correctly handle this case and show all errors before NRE (but not all possible ones)